test/e2e/features/cat-d/02-Failed-journy.feature

```diff
@@ -0,0 +1,50 @@
+@catd
```

Comment:
same typo in file name

Reply:
@Cullima Thank you typo error has been fixed.